src/components/PhotoFrame.tsx

```diff
@@ -53,7 +53,7 @@ const Container = styled.div`
     width: 100%;
     flex-wrap: wrap;
     margin: 0 auto;
-
+    overflow: hidden;
```

Comment:
`overflow: scroll` would be safer bet here.

Reply:
adding`overflow :scroll` adds a scrollbar even when scrollbar is not , `overflow:auto` would be better ?

Reply:
I noticed a thing when i was trying to do this....The of the react `AutoSizer` is providing width and height value greater than that of the window because of which we get a scrollbar even with `overflow:auto`

![image](https://user-images.githubusercontent.com/46242073/133883074-352f2268-e272-4dcc-949f-df8e523bcf43.png)

Reply:
I think it would better if we put css condition to take `min (react_AutoSizer_value ,100%)`

Reply:
I think I understand what the problem is....It doesn't take into account the scrollbar width

Reply:
Ohh The value of autSizer was off by 1 px. 
This  caused a horizontal scrollbar to appear..
and because of that scrollbar the height reduced and a veritical scrollbar appears